bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,323 @@
+import asyncio
+import random
+from datetime import datetime
+from enum import IntEnum
+from typing import Any, Dict, List, Tuple
+
+from aiohttp import ClientSession
+from discord import Embed
+from discord.ext.commands import Bot, Cog, Context, group
+
+from bot.constants import Tokens
+from bot.pagination import ImagePaginator, LinePaginator
+
+# Base URL of IGDB API
+BASE_URL = "https://api-v3.igdb.com/"
+IMAGE_BASE_URL = "https://images.igdb.com/igdb/image/upload/"
+
+HEADERS = {
+    "user-key": Tokens.igdb,
+    "Accept": "application/json"
+}
+
+
+class GameStatus(IntEnum):
+    """Game statuses in IGDB API."""
+
+    Released = 0
+    Alpha = 2
+    Beta = 3
+    Early = 4
+    Offline = 5
+    Cancelled = 6
+    Rumored = 7
+
+
+class AgeRatingCategories(IntEnum):
+    """IGDB API Age Rating categories IDs."""
+
+    ESRB = 1
+    PEGI = 2
+
+
+class AgeRatings(IntEnum):
+    """PEGI/ESRB ratings IGDB API IDs."""
+
+    Three = 1
+    Seven = 2
+    Twelve = 3
+    Sixteen = 4
+    Eighteen = 5
+    RP = 6
+    EC = 7
+    E = 8
+    E10 = 9
+    T = 10
+    M = 11
+    AO = 12
+
+
+class Games(Cog):
+    """Games Cog contains commands that collect data from IGDB."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.http_session: ClientSession = bot.http_session
+
+        # Initialize genres
+        asyncio.get_event_loop().create_task(self._get_genres())
+
+    async def _get_genres(self) -> None:
+        """Create genres variable for games command."""
+        body = "fields name; limit 100;"
+        async with self.http_session.get(BASE_URL + "genres", data=body, headers=HEADERS) as resp:
+            result = await resp.json()
+
+        genres = {genre['name'].capitalize(): genre['id'] for genre in result}
+
+        self.genres = {}
+
+        # Manual check genres, replace sentences with words
+        for genre in genres:
+            if genre == "Role-playing (rpg)":
+                self.genres["Role-playing"] = genres[genre]
+                self.genres["Rpg"] = genres[genre]
+            elif genre == "Turn-based strategy (tbs)":
+                self.genres["Turn-based-strategy"] = genres[genre]
+                self.genres["Tbs"] = genres[genre]
+            elif genre == "Real time strategy (rts)":
+                self.genres["Real-time-strategy"] = genres[genre]
+                self.genres["Rts"] = genres[genre]
+            elif genre == "Hack and slash/beat 'em up":
+                self.genres["Hack-and-slash"] = genres[genre]
+            else:
+                self.genres[genre] = genres[genre]
```

Comment:
Looking over this is very confusing. Correct me if I'm wrong, but you're creating aliases for long genre names.

It would be a lot more maintainable to handle the aliasing logic in a generic loop, test it to make sure it works properly, and then simply define the aliases on the side.
```python
aliases = {
    "Role-playing (rpg)": ["Role-playing", "Rpg"],
}

for original, aliases in aliases.items():
    for alias in aliases:
        genres[alias] = original[original]
```
In this particular case, both aliases are case-less sub-strings of the original. There's room for improvement here in terms of how much the cog could aid the user with look-ups.